SocialFilter/sections/specific.txt

```diff
@@ -2555,7 +2555,7 @@ sciencechannel.com,go.discovery.com,foodnetwork.com,tlc.com##.videoMetadata__soc
 focus.pl##.article-social-media
 innpoland.pl,natemat.pl,mamadu.pl##.fb_share_button
 intersinema.com##.shareBig
-cryptodaily.co.uk,elazighaber23.com,kayisihaber.com,wowkeren.com,gercekfethiye.com,gunisigigazetesi.net##.post-share-area
+elazigonline.com,cryptodaily.co.uk,elazighaber23.com,kayisihaber.com,wowkeren.com,gercekfethiye.com,gunisigigazetesi.net##.post-share-area
```

Comment:
this rule blocks comment section in articles

Reply:
these share buttons are missed
![share](https://user-images.githubusercontent.com/18536328/112734242-97667b00-8f55-11eb-8b9a-dd45bbbd79a1.png)

Reply:
Fixed.

Reply:
I have also added another fix for a similar issue.